src/core/cool_queue.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2024, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include "core/compact_object.h"
+
+namespace dfly {
+
+// Used to store "cold" items before erasing them from RAM.
+//
+class CoolQueue {
+ public:
```

Comment:
We should comment at least somewhere more in detail what it actually means to be cold

Reply:
👍🏼  I will add comments in the next PR